docs/guides/Redux-Integration.md

```diff
@@ -5,6 +5,9 @@ To handle your app's navigation state in redux, you can pass your own `navigatio
 With redux, your app's state is defined by a reducer. Each navigation router effectively has a reducer, called `getStateForAction`. The following is a minimal example of how you might use navigators within a redux application:
 
 ```
+
+import {addNavigationHelpers} from 'react-navigation';
```

Comment:
nit: Add a space before and after `addNavigationHelpers` so it matches the style of the other `import`s in the docs. (See the `import` here: https://reactnavigation.org/docs/guides/screen-tracking)